src/main/java/Weeny/task/Deadlines.java

```diff
@@ -0,0 +1,52 @@
+package weeny.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import weeny.parser.Parser;
+/**
+ * Represents a deadline task with a description and due date/time.
+ */
+public class Deadlines extends Task {
```

Comment:
Should the class name be singular rather than plural since one Deadline is created?